src/shared/WebviewMessage.ts

```diff
@@ -131,7 +134,7 @@ export interface WebviewMessage {
 	apiConfiguration?: ApiConfiguration
 	images?: string[]
 	bool?: boolean
-	value?: number
+	value?: number | boolean | any // Allow number for most messages, boolean for VSCode settings
```

Comment:
Maybe we should leave this one alone for backwards compatibility and add a new entry for settingValue? Otherwise I think we might be weakening existing type checking.

Reply:
> Maybe we should leave this one alone for backwards compatibility and add a new entry for settingValue? Otherwise I think we might be weakening existing type checking.

seems reasonable